### PR TITLE
Change game title

### PR DESCRIPTION
**Pull request by YoomamaFTW**
I think it's an OK name, right?

### CODE REVIEW

**Comment by flextian** [comment]
Idk, I think it needs more spice

**Comment by YoomamaFTW** [comment]
Well we still need to keep enigma in the title.

**Comment by Franccisco** [comment]
I go for Decyphering the Enigma

File: amphibian-alchemists/app/main.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def __init__(self, **kwargs):
         self.APP_DIR = os.path.dirname(os.path.abspath(__file__))
 
     def build(self):
-        self.title = "ENG"
+        self.title = "Enigma Communication"
```
Comment (Franccisco) [inline amphibian-alchemists/app/main.py]:
```suggestion
        self.title = "Decyphering the Enigma"
```

File: amphibian-alchemists/app/kvs/mainscreen.kv [hunk 2]
```diff
@@ -24,12 +24,14 @@
...
             GridLayout:
                 pos_hint: {"center_x": 0.5}
-                size_hint_x: 0.3
+                size_hint_x: 0.6
                 cols: 2
                 Image:
                     source: app.icon
                 Label:
-                    text: app.title
+                    text: "[b]Enigma\nCommunication[/b]"
```
Comment (Franccisco) [inline amphibian-alchemists/app/kvs/mainscreen.kv]:
```suggestion
                    text: "[b]Decyphering\nthe\nEnigma[/b]"
```

**Comment by flextian** [comment]
What about "Elusive Enigma," elusive because you are trying to decipher the enigmas before the allies do it?

**Comment by YoomamaFTW** [comment]
So Elusive Enigmas or Decyphering the Enigma?
For Elusive, since you guys are adding the timer, I can write in the README with something. We can also have like a BOMB sound effect from https://reddit.com/r/shockwaveporn to include if the timer reaches a certain time.

**Pull request merged**